jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6562,11 +6700,15 @@
 							"query": [
 								{
 									"key": "filePath",
-									"value": "D:\\stp\\app\\webapps\\{{HealthCheckWar}}"
+									"value": "D:\\ctp\\app\\webapps\\{{helloWorldWar}}",
```

Comment:
Same thing: we shouldn't be hard coding the path. This should fail when testing against a Linux deployment.